src/ng2-smart-table/components/filter/filter-types/completer-filter.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit } from '@angular/core';
+import { Subject } from 'rxjs/Subject';
+import { CompleterService } from 'ng2-completer';
+
+import { DefaultFilter } from './default-filter';
+import { Column } from '../../../lib/data-set/column';
+
+@Component({
+  selector: 'completer-filter',
+  styles: [`
```

Comment:
Sure, could you tell me why you prefer to move them to a separate file even if they are just a few lines long?